src/diff/index.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Command, flags } from "@oclif/command";
+
+import { findApiChanges, diffRaml } from "./diffProcessor";
+import { NodeDiff } from "./jsonDiff";
+
+export { findApiChanges, diffRaml, RamlDiff } from "./diffProcessor";
+export { NodeDiff } from "./jsonDiff";
+export default class DiffCommand extends Command {
+  static description = `Takes two API spec files as input and outputs the differences.
+By default, a ruleset is applied to determine if changes are breaking. Exit status is:
+  0 - all changes are non-breaking
+  1 - any changes are breaking
+  2 - evaluation could not be completed
+
+The ruleset flag is used to evaluate a custom ruleset in place of the default rules. The diff-only flag disables evaluation against any ruleset.
+`;
+
+  static flags = {
+    // Add --version flag to show CLI version
+    version: flags.version({ char: "v" }),
+    // Add --help flag to show CLI version
+    help: flags.help({ char: "h" }),
```

Comment:
```suggestion
```
These are automatically supported.

Reply:
Version doesn't work without this here and help only work in long form without it.